lib/phoenix_slime/live_view_engine.ex

```diff
@@ -0,0 +1,18 @@
+defmodule PhoenixSlime.LiveViewEngine do
+  @behaviour Phoenix.Template.Engine
+
+  @doc """
+  Precompiles the String file_path into a function definition
+  """
+  def compile(path, _name) do
+    path
+    |> read!
```

Comment:
I thought about making `read!` on `PhoenixSlime.Engine` public and just using it here, but I thought I'd play it safe 🤷‍♂ 😄

Reply:
Would you mind adding the parenthesis on `read!/1`?

Reply:
👍 Done!  Added in `PhoenixSlime.Engine` as well